src/helper_addressbook.py

```diff
@@ -0,0 +1,16 @@
+"""
+Insert value into addressbook
+"""
+
+from bmconfigparser import BMConfigParser
+from helper_sql import sqlExecute, sqlQuery
+
+
+def insert(address, label):
+    """perform insert into addressbook"""
+    queryreturn = sqlQuery(
```

Comment:
This check isn't needed.

Reply:
I have put these checks because sometimes BMConfigParser().addresses() gives an empty list even if there is an address saved in addressbook so in that case we need to check it throught sql query for duplicate addresss

Reply:
> ... sometimes ...

The tests must be as deterministic as possible. They must always succeed. If they sometimes fail, it needs to be fixed properly rather than skipped.

`BMConfigParser().addresses()` are your own addresses, whereas addressbook is a separate list. Maybe your confusing the two?

Reply:
No, I am not confusing actually the problem is when you add the same address twice in Addressbook
![image](https://user-images.githubusercontent.com/33522186/103550275-aac1f700-4ece-11eb-91e4-c9cc23dad2f9.png)
then according to the above method first case will pass but when you again enter the same address in Aaddressbook then it is not failing it is also passing because here
![image](https://user-images.githubusercontent.com/33522186/103550518-07251680-4ecf-11eb-80b9-af92bd82df7a.png)
in insert method BMConfigParser().addresses() always gives empty string even data is present in Addressbook so due to that I have added extra check for checking the address in addressbook

Reply:
do you have any idea why BMConfigParser().addresses() is displaying empty list?

Reply:
So I checked and I mistakenly believed the `addressbook` table has a constraint. It doesn't. So it needs to be fixed.

`BMConfigParser().addresses()` is empty until a new address is generated.